clang/include/clang/AST/Type.h

```diff
@@ -31,6 +31,7 @@
 #include "clang/Basic/PointerAuthOptions.h"
 #include "clang/Basic/SourceLocation.h"
 #include "clang/Basic/Specifiers.h"
+#include "clang/Basic/TargetInfo.h"
```

Comment:
I dropped this include again in https://github.com/llvm/llvm-project/commit/bc3b0fadd5120bd88ed6635583941f7763523c0a because it was causing a significant build time regression. I assume this was a leftover from some earlier iteration of the patch.

Reply:
Shouldn't be necessary, I can make a PR to remove it again.